common/src/main/java/com/wynntils/utils/mc/McUtils.java

```diff
@@ -96,14 +96,6 @@ public static void sendSequencedPacket(PredictiveAction predictiveAction) {
         mc().gameMode.startPrediction(mc().level, predictiveAction);
     }
 
-    /**
-     * Sends the specified command to the server.
-     * @param command The command to send. The leading '/' should not be included.
-     */
-    public static void sendCommand(String command) {
-        mc().getConnection().sendCommand(command);
```

Comment:
Can't you just keep this and let it do `Managers.Command.queueCommand` instead? It will both mean fewer changes, and that there will continue to be a simple to use util function to send commands.

Reply:
Imo that is kind of strange. There was a fairly significant change on wynncraft with the new command ratelimits, and our code should reflect that. It also doesn't make sense that a function that sends commands now just magically queues them, and it is also kind of strange that it is in McUtils of all places, since it is a custom implementation by us.